src/auth/auth.proto

```diff
@@ -72,50 +72,6 @@ message SetConfigurationRequest {
 }
 message SetConfigurationResponse {}
 
-enum ClusterRole {
-    UNDEFINED = 0;
-
-    // SUPER gives access to all APIs and owner on everything in PFS (previous just called admin)
-    SUPER = 1;
-
-    // FS gives Owner on all repos in PFS but not access to manage other aspects of the cluster
-    FS = 2;
-  }
-
-// ClusterRoles reflects any cluster-wide permissions a principal has.
-// A principal can have multiple cluster roles.
-message ClusterRoles {
-  repeated ClusterRole roles = 1;
-}
-
-// Get the current set of principals and roles for the cluster
-message GetClusterRoleBindingsRequest{}
-message GetClusterRoleBindingsResponse{
-  // bindings contains a mapping of principals to cluster roles
-  map<string, ClusterRoles> bindings = 1;
-}
-
-// Set cluster roles for the specified principal. Setting an empty list of roles
-// revokes any roles the principal has.
-message ModifyClusterRoleBindingRequest {
-  string principal = 1;
-  ClusterRoles roles = 2;
-}
-message ModifyClusterRoleBindingResponse {}
-
-// Deprecated. Get the list of cluster super admins.
-message GetAdminsRequest{}
-message GetAdminsResponse{
-  repeated string admins = 1;
-}
-
-// Deprecated. Add and remove users from the set of cluster super admins.
-message ModifyAdminsRequest {
-  repeated string add = 1;
-  repeated string remove = 2;
-}
-message ModifyAdminsResponse {}
```

Comment:
There's no longer a notion of "this group of users are admins" - we could re-add it as "the set of users with a specific role bound on the cluster" but it's not used for anything right now.